build.sbt

```diff
@@ -40,7 +40,8 @@ libraryDependencies ++= Seq(
   "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
   "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
   "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % jacksonVersion,
-  "com.groupon.dse" % "spark-metrics" % "2.0.0"
+  "com.groupon.dse" % "spark-metrics" % "2.0.0",
+  "mysql" % "mysql-connector-java" % "5.1.42"
```

Comment:
Please remove this.
We don't want this library bundled with Metorikku, you can write in the Readme that in order to use the JDBC reader the user must add to the spark submit command the following (mysql for example):
--packages mysql:mysql-connector-java:5.1.42